docs/data/charts/axis/axis.md

```diff
@@ -35,28 +35,25 @@ That's why in most of the demos with single x and y axis you will not see defini
 Those demos use the defaultized values.
 :::
 
-### Axis type
+### Axis type and data
 
-The axis type is specified by its property `scaleType` which expect one of the following values:
+The axis type is specified by its property `scaleType`.
+Axis definition object has a `data` property which expects an array of value coherent with the `scaleType`, as shown in the table below:
 
-- `'band'`: Split the axis in equal band. Mostly used for bar charts.
-- `'point'`: Split the axis in equally spaced points. Mostly used for line charts on categories.
-- `'linear'`, `'log'`, `'sqrt'`: Map numerical values to the space available for the chart. `'linear'` is the default behavior.
-- `'time'`, `'utc'`: Map JavaScript `Date()` object to the space available for the chart.
-
-### Axis data
-
-The axis definition object also includes a `data` property.
-Which expects an array of value coherent with the `scaleType`:
-
-- For `'linear'`, `'log'`, or `'sqrt'` it should contain numerical values
-- For `'time'` or `'utc'` it should contain `Date()` objects
-- For `'band'` or `'point'` it can contain `string`, or numerical values
+| `scaleType` | Description                                                   | Common usage                           | Expected array data type    |
```

Comment:
```suggestion
| scaleType | Description                                                   | Common usage                           | Expected array data type    |
```

or 

```suggestion
| Value | Description                                                   | Common usage                           | Expected array data type    |
```

Looks a bit Weird otherwise to have code in the header

<img width="403" height="163" alt="image" src="https://github.com/user-attachments/assets/3d426592-d4ae-4bf9-bc9e-1fb8c9751734" />